### DISCUSSION TITLE
the cabin remains frozen

### DISCUSSION THREAD

**Discussion by vblacken** [flybywiresim/aircraft#3260] [Help]
the cabin freezes and I can't fly

**Comment by robinottawa** [comment]
Turn up the heat? :D
Is it frozen at startup? Was it unfrozen until something happened? Do you fly ok before, so your system is okay? Sounds like a graphics thing.

**Reply by vblacken** [reply]
like out of nowhere the switches stop moving but they do what they should, but they don't do the animation it's like a bug that lasts a while and after it solves, but today it might fly and when it was fixed it will get back buguear and so all the time

**Reply by robinottawa** [reply]
Try a different version? Sorry, no other ideas unless it's your hardware.

**Reply by vblacken** [reply]
yesterday it worked without problem, but today I update it and I pass that, maybe they fix it in an update, thanks equally
